src/libraries/System.Private.CoreLib/src/System/Reflection/InvokerEmitUtil.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection.Emit;
+
+namespace System.Reflection
+{
+    internal static class InvokerEmitUtil
+    {
+        // If changed, update native stack walking code that also uses this prefix to ignore reflection frames.
+        private const string InvokeStubPrefix = "InvokeStub_";
+
+        internal unsafe delegate object? InvokeFunc(object? target, IntPtr* arguments);
+
+        public static unsafe InvokeFunc CreateInvokeDelegate(MethodBase method)
+        {
+            Debug.Assert(!method.ContainsGenericParameters);
+
+            bool emitNew = method is RuntimeConstructorInfo;
+            bool hasThis = !(emitNew || method.IsStatic);
+
+            // The first element allows the DynamicMethod to be treated as an instance method which is slightly faster than a static.
```

Comment:
```suggestion
            // The first dummy parameter allows the DynamicMethod to be treated as an instance method which is slightly faster than a static.
```

Reply:
Thanks. Updated to `The first parameter is unused and allows`...